### ISSUE TITLE
Tuya Temp/Hum sensor TS0201 ( no screen & LR03 version )

### ISSUE THREAD

**Issue by KipK** [dresden-elektronik/deconz-rest-plugin#5613]
## Device

- Product name: Tuya Smart Zigbee 
- Manufacturer: _TZ3000_bguser20
- Model identifier: TS0201
- Device type : 
  - Sensor

link: https://fr.aliexpress.com/item/1005003507022643.html?spm=a2g0o.9042311.0.0.1b7f6c37iSBltY

## Screenshots

![H8fce06b5cbc04724b5bac89a3ce1309f8](https://user-images.githubusercontent.com/488789/147479344-3dab7fa4-9699-41f8-94bc-a21409da47c9.png)

![sensor_node](https://user-images.githubusercontent.com/488789/147479393-f045ed4b-5714-4a68-a177-a1fc4401d8a8.jpg)

![node](https://user-images.githubusercontent.com/488789/147479362-47c7f219-ff7e-4143-b33f-24f1338e674f.jpg)

### Basic
![basic](https://user-images.githubusercontent.com/488789/147479421-72b278a1-3106-4f7f-b03e-cd6e458d2121.jpg)

### Identify
![identify](https://user-images.githubusercontent.com/488789/147479427-7801bb0a-eceb-48a0-9bab-9d83d1ae9d39.jpg)

### Device Temperature
![temp](https://user-images.githubusercontent.com/488789/147479438-539eff78-ae78-4ffa-99b9-6670d4de74fd.jpg)
![humi](https://user-images.githubusercontent.com/488789/147479449-4e18948b-f55b-499e-b530-b23568ad6ed5.jpg)

### Other clusters that are not mentioned above
![power-conf (2)](https://user-images.githubusercontent.com/488789/147479473-112f1250-d27f-4e46-8cf9-fb735ef31299.jpg)

Not exposed through API yet,  only seen in Deconz GUI.

**Comment by Smanar** [comment]
Hello, this device realy look classic, I think it can work with a DDF file, like here https://github.com/dresden-elektronik/deconz-rest-plugin/issues/5483#issuecomment-981131783

**Comment by KipK** [comment]
Do you have some documentation somewhere about how using this?
I've tried a bit, could have an initial update of values through api then nothing. Seems I miss some details here

**Comment by Smanar** [comment]
If you have initial value but nothing after it s probably because of that https://github.com/dresden-elektronik/deconz-rest-plugin/issues/5483#issuecomment-981757918

Bind is used to tell the device to send value, and the reporting need to be configured to set mworking mode.

**Comment by simontegelid** [comment]
@KipK try this: https://github.com/dresden-elektronik/deconz-rest-plugin/pull/5299#issuecomment-1003585131 I have a bunch of these sensors working with that DDF file.

**Comment by KipK** [comment]
@simontegelid 
Thanks for pointing it. I'm not home for a week, will try that when I'm back

Cheers

**Comment by KipK** [comment]
The provided DDF file works well by renaming the manufacturer to "_TZ3000_bguser20"

Thanks for the fast prompt guys.

**Comment by simontegelid** [comment]
Glad it worked for you! I didn't have to rename the manufacturer, even though bguser20 is in my manufacturer strings on the sensors. Why is it important that they match?

**Comment by KipK** [comment]
You've posted while I was editing my previous one.

As soon as I changed the manufacturer, on Deconz gui the node got a name instead of hex strings.

**Comment by simontegelid** [comment]
Ah. Ok. I got various strange names, so maybe that explains it. However, I managed to rename the sensors in the phoscon app using a workaround for a bug in there. I posted the workaround in the corresponding issue on the matter, but cant find it atm. Will post a link here if I can find it.

**Comment by KipK** [comment]
After some testing, seems It doesn't survive to Phoscon reboot.
Some new FF endpoints mysteriously appears thereafter.

**Comment by simontegelid** [comment]
Hm, ok. I have my status set to Bronze and the "wrong" manufacturer. Survives reboots and reconnects as expected. Simply works like a charm. Would be great to hear from someone more introduced to the internals of Deconz explaining the meaning of a correct manufacturer string and the meaning of various status levels.

**Comment by Smanar** [comment]
On my side, I don't know what happen, but I know 2/3 things
- the issue appear on launch and can be solved using the "gold" status.
- on a previous comment manup had a rough idea where the issue is from
- with the bad "manufacturename" I think the DDF is just not used, easy to check, right clic on the node, "edit DDF", and check the path in the window title.
- Status will be for future, ATM the only impact they have is in the "basic" mode on DDF control.

**Comment by KipK** [comment]
Found what was causing my reboot problem. I was storing the DDF file in /usr/share/dresden.... , so my edited ddf through the GUI ( when I've changed the manufacturer name ) was not saved because of write permission ( It didn't complained while saving it ).
So putting it in ~/config/dresden... , things survive reboot now. My mistake sorry for that.

I'm really happy with those module. Seems they take in count the binding rates parameters as I receive an update each 5mn, seems in sync with the 300s I have setup.
